ethereum-contracts/README.md

```diff
@@ -1,72 +1,61 @@
 
-  - Truffle: `npm install -g truffle` for Mac or go to the install
-  - Testrpc: `npm i -g ethereumjs-testrpc`
-  - Ganache: Download file [here](http://truffleframework.com/ganache/)
+## Project Background
+We are hoping to create a closed system for intra network transfers of cryptocurrency between blockchains, spearheaded by a proof-of-concept which enables secured transactions between Ethereum and Cosmos.
 
-## Instructions
+## Smart Contract Scope
 
-You can test the contracts using Truffle Testrpc in the console or Ganache UI.
+### Goals of the Smart Contracts
+1. Securely implement core functionality of the system such as asset locking and event emission without endangering any user funds. As such, this prototype does not permanently lock value and allows the original sender full access to their funds at any time.
+2. Interface with the Relayer service, which is used by validators to listen for contract events which are signed and submitted to the Cosmos network as proof of transaction.
+3. Successfully end-to-end test the Cosmos Ethereum Bridge, sending Ethereum and ERC20 tokens from Ethereum to Cosmos.
 
-1) **Testrpc**:
-    - `$ cd ethereum-contracts`
-    - Delete *build* folder: `$ rm -r build/`
-    - `$ truffle migrate --reset --compile-all`
-    - `$ testrpc`
-    - `$ truffle test` on a separate tab
+### Non-Goals of the Smart Contracts
+1. Creating a production-grade system for cross-chain value transfers which enforces strict permissions and limits access to locked funds.
+2. Implementing a validator set which enables observers to submit proof of fund locking transactions on Cosmos to Peggy. These features are not required for unidirectional transfers from Ethereum to Cosmos and will be re-integrated during phase two of the project, which aims to send funds from Cosmos back to Ethereum.
+3. Fully gas optimize and streamline operational functionality; ease and clarity of testing has been favored over some gas management and architectural best practices.
 
+## Ethereum Cosmos Bridge Architecture
+Unidirectional Peggy focuses on core features for unidirectional transfers. This prototype includes functionality to safely lock and unlock Ethereum and ERC20 tokens, emitting associated events which provide the Cosmos based Relayer and Oracle modules with the information required to complete unidirectional transfers. Validators witness the events and submit proof in the form of signed hashes to the Cosmos based modules, which are responsible for aggregating and tallying the Validators’ signatures and their respective signing power. The system is managed by the contract's deployer, designated internally as the relayer, a trusted third-party which can unlock funds and return them their original sender. If the contract’s balances under threat, the relayer can pause the system, temporarily preventing users from depositing additional funds.
```

Comment:
What would it mean for the contract's balances to be under threat?

Reply:
'Under threat' refers to any unforeseen circumstances which endanger balances on the contract, such as code insecurity. Pausing locking is a security measure which prevents additional funds from being deposited and therefore at risk in such a scenario.

Reply:
OK.